### PR TITLE
Update plugin parent POM and plugin BOM

### PR DESCRIPTION
**Pull request by basil**
Supersedes #358. Downstream of https://github.com/jenkinsci/metrics-plugin/pull/153. Getting this released will help us cross the recent `jackson2-api` flag day and restore `support-core` to jenkinsci/bom as described in https://github.com/jenkinsci/bom/pull/1011, https://github.com/jenkinsci/metrics-plugin/pull/153, and https://github.com/jenkinsci/bom/pull/1016.

CC @jetersen @Dohbedoh

### CODE REVIEW

File: pom.xml
Comment (basil) [inline pom.xml:118]: https://github.com/jenkinsci/metrics-plugin/pull/153

**Review by jetersen — APPROVED** [review]

**Review by jglick — APPROVED** [review]

**Review by Dohbedoh — APPROVED** [review]

**Comment by basil** [comment]
This is now ready to go from my perspective.

**Comment by jetersen** [comment]
@Dohbedoh a merge and release would really be appreciated, thanks ☺️

**Comment by basil** [comment]
Let us not forget that there is another person with apparent write access who refuses to unblock others by clicking the merge button because reasons.

**Comment by Dohbedoh** [comment]
@basil @jetersen done https://github.com/jenkinsci/support-core-plugin/releases/tag/1158.v9189f64fec8c

**Pull request merged**
